iis-wf/iis-wf-affmatching/src/test/resources/experimentalData/README.markdown

```diff
@@ -0,0 +1,71 @@
+# Rules of matching by hand
+
+A given affiliation is considered to match a given organization if the 
+person that matches them decides so by using all the knowledge they have 
+and by using all accessible sources of information (mainly the Internet).
+
+It is worth noting that the institutions and their departments are not 
+linked in any way in the OpenAire organization database. However, an 
+individual that prepares the quality test data takes the relations 
+between an institution and its departments into account. In particular:
+
+* If a given institution in a given affiliation is a department of a
+  certain organization then it will be matched to this particular
```

Comment:
Michał, could you change all the occurrences of match/matched **to** WITH match/matched **with**? I talked with my english teacher yeasterday about this verb and he said that it should be used with '**with**'.